tracpro/baseline/views.py

```diff
@@ -74,18 +74,40 @@ def get_context_data(self, **kwargs):
 
             region = self.request.region
 
-            baseline_dict = self.object.get_baseline(region=region)
-            context['baseline_dict'] = baseline_dict
+            baseline_dict, baseline_dates = self.object.get_baseline(region=region)
 
             follow_ups, dates = self.object.get_follow_up(region=region)
 
+            """
```

Comment:
If that's not the norm in python, I will go back to single # comments. No worries.